### DISCUSSION TITLE
Usabilité de VTherm (Thermostat HA general d'ailleurs)

### DISCUSSION THREAD

**Discussion by vxav** [jmcollin78/versatile_thermostat#328] [General]
Bonjour,

Tout d'abord mes félicitations a l'auteur de cette intégration qui est excellente! 🥇

Mes questions ici sont assez génériques et ne s'appliquent pas forcément a VTherm en particulier mais j'ai accès a un pool d'afficionados du thermostat dont les retours d'experience seront interessants.

1. Quelle est votre solution pour permettre une utilisation du thermostat a n'importe qui? Par la j'entend par exemple un invité ou simplement la possibilité de le régler sans un telephone ou assistant vocal. 
2. Pour ceux qui n'ont pas les fils pilotes centralisés au tableau electrique, avez vous trouvé des relais qui ne font pas le "clac" a chaque declenchement. Après un rapide google je découvre que la solution serait les "solid state relays" mais ca ne semble pas idéal non plus (chauffe notamment).

Merci et a bientot 👋

**Comment by jmcollin78** [comment]
Hello @vxav ,  You should post this in English to have the maximum chance to get a response.
